packages/page-storage/src/Selection/Raw.tsx

```diff
@@ -27,30 +36,94 @@ function Raw ({ onAdd }: Props): React.ReactElement<Props> {
 
       setValue({
         isValid: u8a.length !== 0,
-        key: compactAddLength(u8a)
+        key,
+        keyAsBytes: compactAddLength(u8a)
       });
     },
     []
   );
 
+  const allStoragePrefixes = useMemo(
+    () => {
+      const palletNames = Object
+        .keys(api.query)
+        .map((value): string => (
+          value
+        ));
+      const allStoragePrefixesNotFlattened = palletNames.map((palletName) => {
+        const pallet = api.query[palletName];
+
+        return Object
+          .keys(pallet)
+          .map((palletCall) => {
+            const { keyPrefix } = pallet[palletCall] as unknown as StorageEntry;
```

Comment:
do you have to do two casts?

Reply:
I've copied it from other file to be honest, not understand yet such TS magic :)